templates/preview-maid.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>PreviewMaid</Name>
+  <Repository>fletchto99/preview-maid</Repository>
+  <Registry>https://hub.docker.com/r/fletchto99/preview-maid</Registry>
+  <Network>bridge</Network>
+  <Shell>sh</Shell>
+  <Privileged>false</Privileged>
+  <Support>https://forums.unraid.net/topic/87798-support-selfhostersnets-template-repository/</Support>
+  <Project>https://github.com/fletchto99/preview-maid</Project>
+  <Overview>Preview maid is a tool to help you find missing thumbnail previews in your plex library. By default it will run upon start and then on a schedule daily at 00:00.</Overview>
+  <Category>Tools:Utilities Status:Stable</Category>
+  <TemplateURL>https://github.com/Kometa-Team/Unraid-Templates/blob/master/ImageMaid.xml</TemplateURL>
```

Comment:
This needs updating. Is this a fork of ImageMaid?

Reply:
Ah good catch. No it's not - I just used their xml as a template